save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/benchmarks/TabMenuBar.kt

```diff
@@ -0,0 +1,58 @@
+package com.saveourtool.save.entities.benchmarks
+
+import com.saveourtool.save.domain.Role
+
+/**
+ * Interface for tab bar in many pages
+ */
+interface TabMenuBar<T> {
```

Comment:
Can it be made `TabMenuBar<T : Enum<T>>`? Then `findEnumElement` and `postfixInRegex` can have default implementation